src/Apps/Conversation/Components/Conversation.tsx

```diff
@@ -0,0 +1,142 @@
+import { Box, Flex, Image, Serif } from "@artsy/palette"
+import { Conversation_conversation } from "__generated__/Conversation_conversation.graphql"
+import React from "react"
+import { createFragmentContainer, RelayProp } from "react-relay"
+import { graphql } from "relay-runtime"
+import { MessageFragmentContainer as Message } from "./Message"
+import { Reply } from "./Reply"
+
+interface ItemProps {
+  item: Conversation_conversation["items"][0]["item"]
+}
+
+const Item: React.FC<ItemProps> = props => {
+  const { item } = props
+  if (item.__typename === "Artwork") {
+    return (
+      <Flex width="350px">
+        <Flex height="auto" alignItems="center" mr={2}>
+          <Image src={item.image.url} width="55px" />
+        </Flex>
+        <Flex flexDirection="column" justifyContent="center">
+          <Serif size="2" weight="semibold">
+            {item.artistNames}
+          </Serif>
+          <Serif italic size="2" color="black60" lineHeight={1.3}>
+            {item.title}, {item.date}
+          </Serif>
+        </Flex>
+      </Flex>
+    )
+  } else if (item.__typename === "Show") {
```

Comment:
Forgot about this, I guess these are inquiries on shows (do we still allow those?)

Reply:
yes, im pretty sure somewhere deep in one of our pages you can still make show inquiries